### PR TITLE
Multicall on batch txs for claiming positions

### PR DESCRIPTION
**Pull request by makiBaraba**

### CODE REVIEW

File: src/constants/network.ts [hunk 1]
```diff
@@ -85,3 +85,5 @@ export const optimismSepolia = {
         },
     },
 } as Chain;
+
+export const GAS_BUFFER_FOR_CLAIM_BATCH = 1.1;
```
Comment (vladanthales) [inline src/constants/network.ts:89]: Please use the existing buffer that we have 1.2 is also ok here.

File: src/pages/Profile/components/OpenClaimableTickets/OpenClaimableTickets.tsx [hunk 2]
```diff
@@ -69,12 +63,12 @@ const OpenClaimableTickets: React.FC<{ searchText?: string }> = ({ searchText })
         networkId,
         claimCollateralIndex,
     ]);
-    const claimCollateralAddress = useMemo(
-        () => getCollateralAddress(networkId, claimCollateralIndex, claimCollateralArray),
-        [networkId, claimCollateralIndex, claimCollateralArray]
-    );
+    // const claimCollateralAddress = useMemo(
```
Comment (vladanthales) [inline src/pages/Profile/components/OpenClaimableTickets/OpenClaimableTickets.tsx:66]: Remove commented code

File: src/pages/Profile/components/OpenClaimableTickets/OpenClaimableTickets.tsx [hunk 3]
```diff
@@ -69,12 +63,12 @@ const OpenClaimableTickets: React.FC<{ searchText?: string }> = ({ searchText })
...
+    //     () => getCollateralAddress(networkId, claimCollateralIndex, claimCollateralArray),
+    //     [networkId, claimCollateralIndex, claimCollateralArray]
+    // );
 
-    const isEth = claimCollateralAddress === ZERO_ADDRESS;
+    // const isEth = claimCollateralAddress === ZERO_ADDRESS;
```
Comment (vladanthales) [inline src/pages/Profile/components/OpenClaimableTickets/OpenClaimableTickets.tsx:71]: Remove commented code

**Review by vladanthales — CHANGES_REQUESTED** [review]

**Review by vladanthales — APPROVED** [review]

File: src/pages/Profile/components/OpenClaimableTickets/OpenClaimableTickets.tsx [hunk 4]
```diff
@@ -110,87 +97,62 @@ const OpenClaimableTickets: React.FC<{ searchText?: string }> = ({ searchText })
...
-                                        toast.update(
-                                            id,
-                                            getErrorToastOptions(t('common.errors.unknown-error-try-again'))
-                                        )
-                                    );
-                                    console.log(e);
-                                }
-                            })
-                        );
+                            const _tx = await sportsAMMV2ContractWithSigner.populateTransaction.exerciseTicket(
```
Comment (lotinasa) [inline src/pages/Profile/components/OpenClaimableTickets/OpenClaimableTickets.tsx:120]: No need for underscore in _tx variable name.

**Review by lotinasa — APPROVED** [review]

**Pull request merged**